notebooks/outlier-treatment.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
<p>Can you change the notebook for our pattern?</p><p>the name could be: 1.3-vse-outliers-clean.ipynb</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/44/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Sure, forgot about that.